low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -89,21 +89,18 @@ def add_args(parser: str, cls: str, name: str):
         parsers.set_defaults(command=cls)
 
     @staticmethod
-    def _send_command(command: str, fail_on_error=True):
+    def _send_command(command, fail_on_error=True):
```

Comment:
rebase on https://github.com/Seagate/cortx-monitor/pull/279, common functions moved to Utility.py